### DISCUSSION TITLE
7.4.3 -> before downgrade DB to 7.4.0

### DISCUSSION THREAD

**Discussion by Berber31** [concourse/concourse#7858] [Help & Support]
@clarafu @taylorsilva @AndoniLarz
To upgrade from 7.4.2 (or 7.4.1) to 7.4.3, the release note 7.4.3 noticed to prior downgrade database to 7.4.0.

So it means a database migration was introced between 7.4.0 and 7.4.1 version ? (so confusing for a patch version...)

In this issue (https://github.com/concourse/concourse/issues/7684#issuecomment-977634807) we faced off, when we upgraded 7.4.0 to 7.4.2 we experimented a latency regression, then we downgraded back to 7.4.0 (without migrating DB from 7.4.2 to 7.4.0) without experimenting "visible" issues. 

What is the way (metrics) we could track to be sure we are not actually facing off regression ?

Do you advise to mandatory migrate back DB to 7.4.0 (with 7.4.2 concourse binary)?

**Comment by taylorsilva** [comment]
> So it means a database migration was introced between 7.4.0 and 7.4.1 version ?

Yes one was introduced to resolve some performance concerns we had for our customers (Pivotal/VMware customers).

> Do you advise to mandatory migrate back DB to 7.4.0 (with 7.4.2 concourse binary)?

You should migrate back or future upgrades to anything after 7.5.0 will result in some migrations being skipped because of the migration number used in 7.4.[1,2] and THEN you'll get a bunch of sql related errors in the web node logs.
